web_external/js/views/body/ImagesView/shims/recolorImageFilters.js

```diff
@@ -0,0 +1,54 @@
+/*globals d3*/
+
+function recolorImageFilters(colorList) { // eslint-disable-line no-unused-vars
```

Comment:
This is sort of a general purpose function that I just copied in from Resonant Lab. I'm sure I'll be bringing more soon (e.g. I rolled my own SVG text wrapping logic, etc) that will also populate the shims directory.

In theory, these things are general-purpose tools and probably *should* be separated into their own repositories + installed via NPM... but they're tiny and that route is definitely overkill at the moment.